index.html

```diff
@@ -980,51 +980,11 @@ <h2>Status</h2>
         <h2>Presentations</h2>
 
         <p>
-Credentials MAY be composed by placing them into a data structure called a
-<a>presentation</a>. A <a>verifiable presentation</a> is a presentation that contains
-<a>verifiable credentials</a> and one or more proofs that are appropriate
-for the presentation. The basic structure of a <a>verifiable presentation</a>
-is provided below:
+Credentials MAY be used to make a <a>verifiable presentation</a>. The presentation
+may be of a credential itself, or of a profile, or of data derived from credentials
+in a format that is cryptographically verifiable.
         </p>
 
-        <pre class="example nohighlight" title="Basic structure of a presentation">
```

Comment:
I'm guessing that the deletion of the majority of the content in this section was an error?

Reply:
No, it was not a mistake. I was proposing that we omit any example of a presentation, because the example shown is SO different from the one that ZKPs have to use. I could submit a PR with a second example of a presentation (this one ZKP-style), but with little in common between the two examples, it's not clear to me what value there would be in pairing them.

Perhaps this needs to be the subject of a different PR.